.github/workflows/metrics.yml

```diff
@@ -39,12 +39,12 @@ jobs:
       - name: Compare performance
         run: |
           echo "## Model Benchmark" >> report.md
-          echo "<details>\n<summary>Show benchmark results</summary>\n" >> report.md
           python tests/metrics/compareMetrics.py >> report.md
       - name: Publish report
         env:
           REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          echo "<details>\n<summary>Model training plots</summary>\n" >> report.md
```

Comment:
I'd prefer to have an option to also collapse the regular metrics - no hard feelings and no blocker for sure 👍

Reply:
Totally, agree, in the current setting there are too many metrics. Lets agree in our weekly meeting on the few key metrics that we always display and make all the others collabsable. I am just afraid that people do not check their metrics if its not visible that their pr reduces performance.